src/components/iconTypes.js

```diff
@@ -0,0 +1,16 @@
+export const HEADER = 'F';
+
+export const CUISINES = {
```

Comment:
```suggestion
export const CuisineIcons = {
```
Would make more sense when importing 
`import { CuisineIcons } from './iconTypes'`
And of course access using
`CuisineIcons.Vegetarian` - Hey, Your'e a vegetarian! 🥗